requirements.txt

```diff
@@ -3,6 +3,5 @@ redis
 parsedatetime
 discord-rewrite==1.0.0
 yarl<1.2
-requests
```

Comment:
this'll break the bot, so remove or wait till my pr is merged.

Reply:
it'll break cuz master version of the sfu and wolf commands use the requests lib

Reply:
No, that's not how it works, please familiarize more. The master version already had the sfu command not using the requests lib, you made that change in #122 and merged it before this branch was even started, and the wolf command uses the wolfram library which has its own requirements.txt. Did you test this on the PR-138 channel and not the async_urban channel? Remember, the branch channels are disabled when the PR is opened to prevent duplication issues.